pkg/kapp/clusterapply/changes_view.go

```diff
@@ -93,16 +89,13 @@ func (v *ChangesView) Print(ui ui.UI) {
 		}
 
 		if resource.IsProvisioned() {
-			condVal := cmdcore.NewConditionsValue(resource.Status())
 
 			row = append(row,
 				// TODO erroneously colors empty value
-				uitable.ValueFmt{V: condVal, Error: condVal.NeedsAttention()},
```

Comment:
Now that we are not using `ConditionsValue` anywhere, we could also remove [this](https://github.com/vmware-tanzu/carvel-kapp/blob/develop/pkg/kapp/cmd/core/conditions_value.go) file.

Reply:
+1. There is no point in keeping this file now.

Reply:
Removed